packages/form-data/test/form-data.spec.js

```diff
@@ -259,4 +259,28 @@ export const test = (test) => {
       "rename.md"
     );
   });
+
+  test("Should allow passing a form element",  () => {
```

Comment:
Let's drop the jsdom dev dependency and just make our own mock `HTMLFormElement` that's got all the right methods/getters/etc. that we use.